bundles/ui/org.eclipse.smarthome.ui/src/main/java/org/eclipse/smarthome/ui/internal/items/ItemUIRegistryImpl.java

```diff
@@ -433,7 +433,7 @@ protected String formatUndefined(String formatPattern) {
         try {
             return String.format(undefinedFormatPattern, "-");
         } catch (Exception e) {
-            logger.warn("Exception while formatting undefined value [sourcePattern={}, targetPattern={}, {}]",
+            logger.warn("Exception while formatting undefined value [sourcePattern={}, targetPattern={}, exception= ",
```

Comment:
Now the format is broken: The closing bracket is missing and also the string placeholder for the exception message is missing. Add it again and provide a `e.getMessage()` instead of the full exception.